docs/CONTRIBUTING.md

```diff
@@ -60,90 +60,12 @@ please create an issue on the GitHub repository:
   or in a comment to the issue.
 
 ## Building the documentation locally
+ * For more information on this:  
+      - [read here](docs/builddocslocally.md)
```

Comment:
We generally avoid using strings like "read here" or "here" for links.  Better to use the title of the page being referenced.